Jenkinsfile

```diff
@@ -23,6 +23,8 @@ pipeline {
         DISABLE_DOWNLOAD_PROGRESS_OPTS = '-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn '
         LINUX_MVN_RANDOM = '-Djava.security.egd=file:/dev/./urandom'
         COVERAGE_EXCLUSIONS = '**/test/**/*,**/itests/**/*,**/*Test*,**/sdk/**/*,**/*.js,**/node_modules/**/*,**/jaxb/**/*,**/wsdl/**/*,**/nces/sws/**/*,**/*.adoc,**/*.txt,**/*.xml,**/platform-solr-server-standalone/**/*'
+        OWASP_ANALYZER_OPTS = '-DarchiveAnalyzerEnabled=false -DopensslAnalyzerEnabled=false -DnspAnalyzerEnabled=false -DnuspecAnalyzerEnabled=false -DassemblyAnalyzerEnabled=false'
```

Comment:
From @mackncheesiest : 
Now since we're upgrading/modifying the plugin anyway, we might as well disable the analyzers in the plugin config so that we don't have a maintenance issue between pipeline jobs and legacy jobs

i.e. see alliance's config